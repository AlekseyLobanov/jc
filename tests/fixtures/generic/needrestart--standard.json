{"version":"2.1","running_kernel_version":"3.19.3-tl1+","expected_kernel_version":"3.19.3-tl1+","kernel_status":1,"container":"LXC web1","session":["metabase @ user manager service","root @ session #28017"],"service":["systemd-journald.service","systemd-machined.service"]}
