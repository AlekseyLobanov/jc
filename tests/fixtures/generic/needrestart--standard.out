NEEDRESTART-VER: 2.1
NEEDRESTART-KCUR: 3.19.3-tl1+
NEEDRESTART-KEXP: 3.19.3-tl1+
NEEDRESTART-KSTA: 1
NEEDRESTART-SVC: systemd-journald.service
NEEDRESTART-SVC: systemd-machined.service
NEEDRESTART-CONT: LXC web1
NEEDRESTART-SESS: metabase @ user manager service
NEEDRESTART-SESS: root @ session #28017
